libmachine/ssh/client.go

```diff
@@ -65,7 +65,6 @@ const (
 var (
 	baseSSHArgs = []string{
 		"-F", "/dev/null",
-		"-o", "BatchMode=yes",
 		"-o", "PasswordAuthentication=no",
```

Comment:
Yes it does, my intention was only to allow prompts for ssh key passphrases. Do you think password auth should be supported too? I'm not sure about the impacts.